checker/src/main/java/org/checkerframework/checker/index/qual/EnsuresLTLengthOf.java

```diff
@@ -78,4 +80,20 @@
     @JavaExpression
     @QualifierArgument("offset")
     String[] offset() default {};
+
+    /** An inner annotation which makes {@link EnsuresLTLengthOf} annotation repeatable */
+    @Documented
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
+    @PostconditionAnnotation(qualifier = LTLengthOf.class)
```

Comment:
Does this work?

The documentation of `@ConditionalPostconditionAnnotation` says:

> The annotation E that is meta-annotated as ConditionalPostconditionAnnotation must
> have an element called `expression` that is an array of Strings of the same format
> and with the same meaning as the value `expression` in `EnsuresQualifierIf`.

and that is not true of this annotation.  So either the definition of this annotation, or the documentation of `@ConditionalPostconditionAnnotation`, must be changed.

Reply:
It is annotated with `@PostconditionAnnotation`, not with `@ConditionalPostconditionAnnotation`.

Reply:
Thanks for the correction.  The comment still applies.

Reply:
You still have not addressed this comment, one week later (and after you have multiple times told me that the pull request is ready to review).  Please address all comments rather than making me find the ones you ignored.

Reply:
@mernst The documentation of `@PostconditionAnnotation` says-
>The annotation E that is meta-annotated as PostconditionAnnotation must have an element called  `value` that is an array of Strings of the same format and with the same meaning as the value `expression` in `EnsuresQualifier`

The annotation `@EnsuresLTLengthOf` has an array `value` of Strings which is true for this annotation. Is something else wrong?

Reply:
This comment thread is associated with the annotation `EnsuresLTLengthOf.List`.
`EnsuresLTLengthOf.List` is meta-annotated with `@PostconditionAnnotation` (the line of code this comment thread is associated with).  That is in violation of the documentation of `@PostconditionAnnotation`.

This is what I said in the very first comment in this comment thread, so I feel that we are going in circles.  Do you see that `@PostconditionAnnotation`'s documentation says it should only be applied to annotations whose `value` field is a array of strings, but here it is applied to an annotation (namely, `EnsuresLTLengthOf.List`) whose `value` field is not an array of strings?

Reply:
@mernst I have edited the documentation of this. Please give a feedback on that.

Reply:
Your edits leave the documentation wrong.  Perhaps I should take over this pull request, since you are having trouble completing it.